test/Control.t.sol

```diff
@@ -189,4 +220,34 @@ contract Control is Test {
         // Check that the stable token address was updated correctly
         assertEq(address(blueberryStaking.stableAsset()), address(token));
     }
+
+    function testChangeRewardAmount() public {
+        vm.startPrank(owner);
+
+        uint256 rewardAmount = REWARD_DURATION * 1e18;
+        // Set reward amounts for existing bTokens
+        uint256[] memory amounts = new uint256[](3);
+        amounts[0] = rewardAmount;
+        amounts[1] = rewardAmount;
+        amounts[2] = rewardAmount;
+
+        // Set initial Reward Amounts
+        blueberryStaking.modifyRewardAmount(existingBTokens, amounts);
+
+        uint256 length = existingBTokens.length;
+        for (uint256 i=0; i < length; ++i) {
+            uint256 rewardRate = blueberryStaking.rewardRate(existingBTokens[i]);
+            assertEq(rewardRate, 1e18);
+        }
+
+        skip(90 days);
+
+        // Add new Reward Amounts in the middle of reward period
+        blueberryStaking.modifyRewardAmount(existingBTokens, amounts);
+
+        for (uint256 i=0; i < length; ++i) {
+            uint256 rewardRate = blueberryStaking.rewardRate(existingBTokens[i]);
+            assertEq(rewardRate, 1e18);
+        }
```

Comment:
The comment suggests that this block tests adding reward amounts in the middle of a reward period, but as far as I can tell it does not do that. It waits 90 days, whereas the reward period is finished after 14 days.

This is a good test, but we should also add a test that introduces extra rewards in the middle of the period (e.g., 7 days after the previous update). We'd expect the `rewardRate` to increase by 50% in that case.